corehq/apps/accounting/models.py

```diff
@@ -704,6 +705,19 @@ def get_version(self):
         except SoftwarePlanVersion.DoesNotExist:
             return None
 
+    def at_max_domains(self):
+        # If no domain limit is set, return True
```

Comment:
Looks like this comment says the opposite.  Since this code is pretty self-documenting, you can probably do without the comment.